frontend/server/src/main/java/org/pytorch/serve/ModelServer.java

```diff
@@ -104,7 +111,10 @@ public void startAndWait()
             throws InterruptedException, IOException, GeneralSecurityException,
                     InvalidSnapshotException {
         try {
-            List<ChannelFuture> channelFutures = start();
+            List<ChannelFuture> channelFutures = startRESTserver();
```

Comment:
it is better to be configurable: REST/gRPC/both

Reply:
@lxning: Logged following enhancement ticket to track this: #799. This is not a blocker item for the 0.3.0 release.